### PR TITLE
Resolved #193: add tui unit test

### LINKED ISSUE #193: [UnitTest]: unittest on tui

### PR DESCRIPTION
**Pull request by chhsiao90**
Coverage up up~~

### CODE REVIEW

**Comment by mike820324** [comment]
+5.4%
Orz

File: microproxy/viewer/tui.py [hunk 1]
```diff
@@ -60,11 +61,17 @@ def _fold_path(self, path):
         max_width = self.terminal_width - 16
         return path if len(path) < max_width else path[:max_width - 1] + "..."
 
+    def _format_port(self, scheme, port):
```
Comment (mike820324) [inline microproxy/viewer/tui.py:64]: good work. The 80/443 shouldn't been shown when it is in the right scheme.

**Review by mike820324 — APPROVED** [review]

**Comment by mike820324** [comment]
Still WIP?

**Comment by chhsiao90** [comment]
No, ready for merge! Wait for your review

**Comment by mike820324** [comment]
Merged thanks.

**Pull request merged**